tools/dockerfile/interoptest/grpc_interop_go1.x.current_version

```diff
@@ -0,0 +1,2 @@
+us-docker.pkg.dev/grpc-testing/testing-images-public/grpc_interop_go1.x:latest
```

Comment:
TODO(arjan-bal): Replace this with the digest once the image is pushed to the registry.